resources/github-actions.md

```diff
@@ -45,6 +45,61 @@ There is a link provided at the end of every C++ and Python build job where the
 
 ![](/assets/images/downloads.png)
 
+## Using CI Artifacts in Other PRs
+
+For changes that cross library boundaries, it may be necessary to test a pull request to one library with changes from a pull request to another library.
+Consider the overall RAPIDS dependency graph when testing.
+For example, if you are testing artifacts from an RMM PR rmm#A in cuML, you probably also need to create a cuDF PR cudf#B that uses the artifacts from rmm#A, and then your cuML test PR will need to include the artifact channels for both rmm#A and cudf#B.
+
+To do this, it is necessary to download CI artifacts (described in the above section) from one library during the CI workflow of another library.
+First, determine the pull request number(s) to be tested from the other library.
+Then, fetch the CI artifacts from the other library's pull request and use them when building and testing.
+The example code below demonstrates building and testing with conda packages from other library PRs.
+Replace the pull request numbers and library names as needed.
+
+**Example 1:** Building `libcuml` (C++) using `librmm`, `libraft`, `libcumlprims_mg` PR artifacts.
+
+```sh
+# ci/build_cpp.sh
+
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1095 cpp)
+LIBRAFT_CHANNEL=$(rapids-get-pr-conda-artifact raft 1388 cpp)
+LIBCUMLPRIMS_CHANNEL=$(rapids-get-pr-conda-artifact cumlprims_mg 129 cpp)
+
+# Build library packages with the CI artifact channels providing the updated dependencies
+
+rapids-mamba-retry mambabuild \
+    --channel "${LIBRMM_CHANNEL}" \
+    --channel "${LIBRAFT_CHANNEL}" \
+    --channel "${LIBCUMLPRIMS_CHANNEL}" \
+    conda/recipes/libcuml
+```
+
+**Example 2:** Testing `cudf` (Python) using `librmm`, `rmm`, and `libkvikio` PR artifacts.
+
+```sh
+# ci/test_python_common.sh
+
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1223 cpp)
+RMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1223 python)
+LIBKVIKIO_CHANNEL=$(rapids-get-pr-conda-artifact kvikio 224 cpp)
+
+# Install library packages with the CI artifact channels providing the updated dependencies for testing
+
+rapids-mamba-retry install \
+  --channel "${CPP_CHANNEL}" \
+  --channel "${PYTHON_CHANNEL}" \
+  --channel "${LIBRMM_CHANNEL}" \
+  --channel "${LIBKVIKIO_CHANNEL}" \
+  --channel "${RMM_CHANNEL}" \
+  cudf libcudf
+```
+
+Note that the custom channel for PR artifacts is needed in the build scripts _and_ the test scripts, for C++ _and_ Python.
+If building/testing a Python package that depends on a C++ library, it is necessary to use PR artifacts from that C++ library and not just Python (e.g. if testing `rmm` artifacts, you must use the corresponding `librmm` CI artifacts as well as `rmm`).
+In some repos, the `test_python.sh` is quite complicated with multiple calls to conda/mamba.
+We recommend that the Python and C++ artifact channels should be added to every call "just in case."
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💡 suggestion:** ‏ For clarity:

```suggestion
We recommend that the Python and C++ artifact channels should be added to every call of `rapids-mamba-retry`/`rapids-conda-retry` "just in case."
```